src/unstract/sdk/file_storage/fs_provider.py

```diff
@@ -0,0 +1,10 @@
+import enum
+
+
+class FileStorageProvider(enum.Enum):
+    Azure = "azure"
```

Comment:
UPPER_CASE is prefered for enum names.  would be better to validate the enum by to upper.

ref: https://docs.python.org/3/howto/enum.html#:~:text=Note%20Case%20of,in%20our%20examples.

Reply:
Done.